### PR TITLE
Add protected branch name description

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

**Review by nanguanlin6 — APPROVED** [review]

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -2300,6 +2300,7 @@ settings.dismiss_stale_approvals_desc = When new commits that change the content
 settings.require_signed_commits = Require Signed Commits
 settings.require_signed_commits_desc = Reject pushes to this branch if they are unsigned or unverifiable.
 settings.protect_branch_name_pattern = Protected Branch Name Pattern
+settings.protect_branch_name_pattern_desc = "Protected branch name patterns. See github.com/gobwas/glob documentation for pattern syntax. Examples: main, release/**"
```
Comment (delvh) [inline options/locale/locale_en-US.ini]: I think you mean
```suggestion
settings.protect_branch_name_pattern_desc = "Protected branch name patterns. See <a href="github.com/gobwas/glob">the documentation</a> for pattern syntax. Examples: main, release/**"
```

Comment (delvh) [reply options/locale/locale_en-US.ini]: And preferably, extract the link location into a variable.

Comment (lunny) [reply options/locale/locale_en-US.ini]: I copied them from other translation. It seems all of them doesn't use a tag but the link is there in the UI.

Comment (delvh) [reply options/locale/locale_en-US.ini]: Sounds like a bug to me.
And nothing we should promote doing.
We've often enough seen the problems of hardcoded links in the translation already…

I mean, I can somewhat guess that the browsers see the URL and say, ah, they want to link somewhere!
But still, it is technically buggy behavior as you cannot guarantee that every browser treats it as a link.

**Review by silverwind — APPROVED** [review]

**Review by delvh — APPROVED** [review]
I'd still prefer it if we would inject the link dynamically, but the current approach is fine too.

**Pull request merged**